stories/__tests__/SQFormTextarea.stories.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormTextarea.stories';
+
+const {
+  Default: SQFormTextarea,
+  WithValidation: SQFormTextareaWithValidation
+} = composeStories(stories);
+
+describe('SQFormTextField Tests', () => {
+  describe('Text Field Only', () => {
+    it('should render a label and text box', () => {
+      render(<SQFormTextarea size="auto" />);
+
+      //There's 2 labels but one has a parent with aria-hidden
+      const label = screen.getAllByText(/textarea/i)[0];
```

Comment:
It's annoying that Mui does this and doesn't give them proper roles.. 🙄

What we could do, instead of relying on the order of the returned array of elements, is find the element in the array that has the `<label>` parent element.
```js
const label = screen
        .getAllByText(/textarea/i)
        .find(item => item.parentElement.tagName === 'LABEL');
```
Kind of hacky and tests implementation details, but it at least gives us the confidence that we're always targeting the right "label"